controllers/render.js

```diff
@@ -0,0 +1,15 @@
+const createError = require("http-errors");
+
+const MethodBlock = require("../models/MethodBlock");
+const InputBlock = require("../models/InputBlock");
+
+exports.renderBlocks = async (req, res, next) => {
```

Comment:
이 컨트롤러도 일관성을 위해 마지막 줄에서 default export 하는 건 어떨까요? 😊
```suggestion
module.exports = renderBlocks;
```

Reply:
감사합니다 용섭님!
해당 내용 반영하여 수정 작업 진행하였습니다.